faction/index.html

```diff
@@ -0,0 +1,257 @@
+        <a href="#" class="btn">Continue Reading</a>
+        
+      </section>
+      <hr>
+      <section class="arcu-container">
+        <header class="arcu-head">
+            <h2>Arcu dolore</h2>
+            <p>Etiam quis viverra lorem in semper lorem.</p>
+            <span>September 15, 2016</span>
+        </header>
+        <hr>
+        <article class="arcu-content">
+            <p>Sed nisl arcu euismod sit amet nisi euismod sed cursus elementum ipsum.</p>
+            <span>September 15, 2016</span>
+        </article>
+        <hr>
+        <article class="arcu-content">
+            <p>Arcu vivamus quis venenatis orci lorem.</p>
+            <span>September 15, 2016</span>
+        </article>
+        <hr>
+        <article class="arcu-content">
+            <p>Veroeros aliquam. Lorem ipsum dolor sit amet nullam dolore magna.</p>
+            <span>September 15, 2016</span>
+        </article>
+        <hr>
+        <article class="arcu-content">
+            <p>Mauris neque quam, fermentum ut nisl vitae</p>
+            <span>September 15, 2016</span>
+        </article>
+        <hr>
+        <article class="arcu-content">
+            <p>Convallis maximus nisl sed mattis nunc id lorem euismod amet placerat.</p>
+            <span>September 15, 2016</span>
+        </article>
+        <hr>
+        <article class="arcu-content">
+            <p>Sed nisl arcu euismod sit amet nisi euismod sed cursus elementum ipsum.</p>
+            <span>September 15, 2016</span>
+        </article>
+        <a href="#" class="btn">More Posts</a>
+      </section>
+      <hr>
+      <section class="about-container">
+        <article class="about-content">
+        <h3>About Us</h3>
+        <p class="para">Morbi mattis mi consectetur tortor elementum, varius pellentesque velit convallis. Aenean tincidunt lectus auctor mauris maximus, ac scelerisque ipsum tempor. Duis vulputate ex et ex tincidunt, quis lacinia velit aliquet. Duis non efficitur malesuada.</p>
+        <p>Sagittis felis ac sagittis semper. Curabitur purus leo donec vel dolor at arcu tincidunt bibendum. Interdum et malesuada fames ac ante ipsum primis.</p>
+        <a href="#" class="btn">Learn More</a>
+    </article>
+    <article class="about-items">
+        <ul class="items">
+            <li><img src="https://fastly.picsum.photos/id/941/536/354.jpg?hmac=8dGR0ZI6HvkSkMKPgven760t2RIple7hEB1QHsfzkaU" alt="picsum photo"
+                class="contact-img"
+                ></li>
+            <li><strong>John Doe</strong><p>Lorem semper magna etiam</p></li>
+        </ul>
+        <ul class="items">
```

Comment:
que: why are u used  ul elements again?
sug: use one ul elements and inside it put all the li elements.